### ISSUE TITLE
What is norecurse?

### ISSUE THREAD

**Issue by secretrobotron** [tosdr/tosback2#65]
I often see `<norecurse name="arbitrary" />`, but I'm not sure what it does. Could we put a section in the README that explains it?

**Comment by michielbdejong** [comment]
Yes, we should! I just read the code, it disables https://www.gnu.org/software/wget/manual/html_node/Recursive-Download.html
